README.md

```diff
@@ -88,6 +88,7 @@ Within the following table, we summarized the current NNI capabilities, we are g
            <li><a href="docs/en_US/TrialExample/GbdtExample.md">Auto-gbdt</a></li>
            <li><a href="docs/en_US/TrialExample/Cifar10Examples.md">Cifar10-pytorch</li></a>
            <li><a href="docs/en_US/TrialExample/SklearnExamples.md">Scikit-learn</a></li>
+           <li><a href="examples/trials/efficientnet">EfficientNet</a></li>
```

Comment:
Why not put the doc under the  "docs/en_US/TrialExample/"?

Reply:
If the docs are to be put under `TrialExample`, they are intended to be read on readthedocs, which means every link on README should be a link to readthedocs website, instead of a link to a markdown on Github. Refactoring involves many changes and should be open to discussion.

Reply:
If you concern that you cannot support markdown and readthedocs at the same time, you could put in the path but do not link to readthedocs in [rst file](https://github.com/microsoft/nni/blob/master/docs/en_US/examples.rst).

Reply:
@ultmaster please put it under docs/